rbkit-lib/cpu/cpumapping.h

```diff
@@ -0,0 +1,28 @@
+#include <QString>
+
+#ifndef MAPPING
+#define MAPPING
+
+namespace RBKit {
```

Comment:
Are we already using such a namespace ? Asking because we've been using "Rbkit" everywhere and not "RBKit" in rbkit-server, blogs, talks etc.